HUDKit/Classes/Progress/HUDProgressViewController.swift

```diff
@@ -67,13 +67,16 @@ public class HUDProgressViewController: UIViewController {
         }
     }
 
+    public var visualEffect: UIBlurEffectStyle?
+
     // MARK: - Initializer
 
-    public init(customView: UIView?, status: String?) {
+    public init(customView: UIView?, status: String?, visualEffect: UIBlurEffectStyle = .Dark) {
```

Comment:
I would prefer that this initializer receive an instance of `UIVisualEffect` rather than `UIBlurEffectStyle` for flexibility.